cmd/frontend/graphqlbackend/search_results.go

```
@@ -1778,7 +1778,10 @@ func (r *searchResolver) doResults(ctx context.Context, args *search.TextParamet
 		wg.Add(1)
 		goroutine.Go(func() {
 			defer wg.Done()
-			_ = agg.DoSearch(ctx, job, args.Repos, args.Mode)
+			err := agg.DoSearch(ctx, job, args.Repos, args.Mode)
+			if err != nil {
+				log15.Warn("agg.DoSearch failed", "error", err.Error())
```

This error was previously being swallowed, meaning we didn't have visibility into the error text

---

The error isn't really swallowed, but handled by `agg.Error`. That's perhaps a better place to log errors since all search jobs would make use of it?